src/neuromaps_nhp/tests/test_volumetric_transform.py

```diff
@@ -0,0 +1,54 @@
+from pathlib import Path
+from neuromaps_nhp.volumetric_transformation import _vol_to_vol
+import nibabel as nib
+import subprocess
+
+
+def _extract_res(nii_file: Path):
+    """Extract voxel spacing from a NIfTI file using wb_command."""
+    cmd = ["wb_command", "-file-information", str(nii_file)]
+    result = subprocess.run(cmd, capture_output=True, text=True)
+    if result.returncode != 0:
+        raise RuntimeError(f"wb_command failed: {result.stderr}")
+
+    for line in result.stdout.splitlines():
+        if "spacing:" in line.lower():
+            spacing_values = line.split(":", 1)[1].replace("mm", "").strip()
+            return tuple(float(x.strip()) for x in spacing_values.split(","))
+    raise ValueError(f"cannot determine resolution {nii_file}")
+
+
+def test_vol_to_vol():
```

Comment:
I would suggest either creating a `Class` that holds all the test for `vol_to_vol`, or have separate functions for each test. That way you can test each test case separately and if something fails, it can be identified quickly. 

Happy to chat more about this!